Update_user.css

```diff
@@ -20,36 +27,68 @@ body{
 
 nav{
 	width:100%;
-	background-color:white;
+	background-color:#0C59CF;
 	overflow:auto;
 }
 
 ul{
 	padding:0;
-	margin:0 0 0 150px;
+	margin:0 0 0 30px;
 	list-style: none;
 }
-
+/* Set up position for the text in nav to the right */
 li{
 	float:right;
 }
-
+/* Set up position for the logo */
 .logo img{
 	position:absolute;
 	margin-top:-20px;
 }
 
+/* Set up Drop down menu for */
 nav a{
 	display:block;
 	width:100px;
 	padding: 20px 15px;
 	text-decoration:none;
-	font-family:Arial;
-	color:black;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:bold;
+	color:white;
 	text-align: center;
 }
 
+nav a:hover{
```

Comment:
I added a hover function in the like so that when the user hover on the link. Then the background will will turn to black and it change the background color after 0.5 second.